styleguide/source/_patterns/01-atoms/media/icons/svg/icon-triangle.twig

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please run` gulp minifyIcons` to remove extraneous svg stuff

Reply:
running `gulp minifyIcons` has no effect on the svg